ext/reflection/php_reflection.stub.php

```diff
@@ -696,7 +696,7 @@ private function __construct() {}
 }
 
 /** @not-serializable */
-final class ReflectionAttribute implements Reflector
+class ReflectionAttribute implements Reflector
```

Comment:
`ReflectionAttribute` exists since PHP 8.0, can we target this version as per https://bugs.php.net/bug.php?id=81474#1632553296 comment?